app/services/email_flowdock_service.rb

```diff
@@ -8,7 +8,7 @@ def initialize(user)
 
   def send_otp
     token = @user.most_recent_token
-    FlowdockNotification.otp(token).deliver # deliver_later once on Rails 4.2
+    FlowdockNotificationJob.perform_later(token)
   end
 
 end
```

Comment:
@shivani just revert this commit to say .deliver_later and remove the previous active_job